src/utils/token.js

```diff
@@ -0,0 +1,36 @@
+import Cookies from "js-cookie";
+
+const expireTime = 365;
+
+export const checkTokens = () => (
+    Cookies.get("accessToken") || Cookies.get("refreshToken")
+);
+
+export const clearTokens = () => {
+    Cookies.remove("accessToken");
+    Cookies.remove("refreshToken");
+};
+
+export const getAccessToken = () => (
+    Cookies.get("accessToken")
+);
+
+export const getRefreshToken = () => (
+    Cookies.get("refreshToken")
+);
+
+export const refreshAccessToken = (accessToken) => {
```

Comment:
How is this function different from [setAccessToken](https://github.com/tolstoyevsky/cusdeb-web-client/pull/111/files#diff-ab13ae8ecb6a1a23cd4274db35c205eb7d97776d200cc7320f9582d7c8a82644R30)?